### DISCUSSION TITLE
Nebula 3.0.0 is out

### DISCUSSION THREAD

**Discussion by lcaron** [EclipseNebula/nebula#512] [Announcements]
I would like to inform you that Nebula 3.0.0 is now alive 😀

* Download page: https://www.eclipse.org/nebula/downloads.php
* N & N : https://wiki.eclipse.org/Nebula/Releases/3.0.0/NaN

The following people contributed to this release (alphabetical order) : Dirk Fauth, Jolkdarr, Laurent Caron.

Thanks to everybody for your work and support, it's great to be part of the Nebula Project.

This release is the first one that no longer runs with JDK 8 (see https://github.com/eclipse/nebula/issues/408).

Ready now for Nebula 3.1.0, which is scheduled on June 30th.

**Comment by wimjongman** [comment]
Thanks, Laurent!

**Comment by fipro78** [comment]
Thanks Laurent!

Wim Jongman ***@***.***> schrieb am Do., 9. März 2023, 17:50:
